<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexSelct</class>
 <widget class="QWidget" name="IndexSelct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于机器学习的电火花小孔加工穿透检测</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK JP</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>放电加工状态</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="mainplot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>650</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="feedrateplot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>650</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currModelLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>穿透检测模型</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>650</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="brekoutLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>650</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>未穿透</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>690</y>
     <width>230</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modelNameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>650</y>
     <width>230</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前用于穿透检测的模型</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>模拟信号文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>690</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>观察到穿透</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
